### PR TITLE
Change klog info messages after a strategy is exited into error messages

### PR DESCRIPTION
**Pull request by ingvagabund**
Builds on top of https://github.com/kubernetes-sigs/descheduler/pull/407

### CODE REVIEW

File: pkg/descheduler/strategies/pod_antiaffinity.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func checkPodsWithAntiAffinityExist(pod *v1.Pod, pods []*v1.Pod) bool {
 			namespaces := utils.GetNamespacesFromPodAffinityTerm(pod, &term)
 			selector, err := metav1.LabelSelectorAsSelector(term.LabelSelector)
 			if err != nil {
-				klog.Infof("%v", err)
+				klog.InfoS("Unable to convert LabelSelector into Selector", "err", err)
```
Comment (damemi) [inline pkg/descheduler/strategies/pod_antiaffinity.go:115]: Should this be `ErrorS`?

Comment (ingvagabund) [reply pkg/descheduler/strategies/pod_antiaffinity.go:115]: Yeah

**Review by damemi — APPROVED** [review]
/approve

**Review by seanmalloy — APPROVED** [review]

**Pull request merged**